frontend/views/containers/chatroom/MessageBase.vue

```diff
@@ -1,5 +1,5 @@
 <template lang='pug'>
-.c-message(
+.c-message.force-motion(
```

Comment:
Message highlight animation still needs to happen regardless of `reduced-motion` setting.